src/commands/apps/show.ts

```diff
@@ -0,0 +1,33 @@
+// apps show command
+
+import { AppCommand, CommandArgs, CommandResult, help, success, failure, ErrorCodes } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { reportApp } from "./lib/format-app";
+import { MobileCenterClient, models, clientCall } from "../../util/apis";
+
+const debug = require("debug")("mobile-center-cli:commands:apps:create");
+import { inspect } from "util";
+
+@help("Get the details of an app")
+export default class AppShowCommand extends AppCommand {
+  constructor(args: CommandArgs) {
+    super(args);
+  }
+
+  async run(client: MobileCenterClient): Promise<CommandResult> {
+    const app = this.app;
+
+    const appDetails = await out.progress("Getting app details ...", clientCall<models.AppResponse>(cb => client.account.getApp(app.appName, app.ownerName, cb)));
+
+    switch ((appDetails as any).error.code as string) {
+      case "BadRequest":
+        return failure(ErrorCodes.Exception, "the request was rejected for an unknown reason");
+      case "NotFound":
+        return failure(ErrorCodes.NotLoggedIn, `the app "${app.identifier}" could not be found`);
```

Comment:
I don't think this is the right return code - this is the "user tried to do something that requires them to be logged in but they aren't" code.